src/base/config.lua

```diff
@@ -49,10 +49,14 @@
 		local bundlepath = ""
 
 		if cfg.system == p.MACOSX and kind == p.WINDOWEDAPP then
-			bundlename = basename .. ".app"
+			bundlename = basename .. extension
 			bundlepath = path.join(bundlename, "Contents/MacOS")
 		end
 
+		if cfg.system == p.MACOSX and kind == p.CONSOLEAPP then
```

Comment:
Same comment as before, is there any reason why we're being so hostile towards `ConsoleApp`?

Reply:
It was the same reason I wrote above.
However, it is not good as it fails the test in this part.